drivers/hmis/app/graphql/types/hmis_schema/income_benefit.rb

```diff
@@ -15,9 +15,11 @@ def self.configuration
     field :id, ID, null: false
     field :enrollment, HmisSchema::Enrollment, null: false
     field :client, HmisSchema::Client, null: false
-    field :user, HmisSchema::User, null: false
+    field :user, HmisSchema::User, null: true
     hud_field :information_date
-    # hud_field :income_from_any_source
+
+    # Income
+    hud_field :income_from_any_source, HmisSchema::Enums::Hud::NoYesReasonsForMissingData
```

Comment:
Is it worth putting something together like `yes_no_missing_field` for these that have yes, no missing, reasons?

Reply:
Hmm maybe. We need `yes_no_missing_field` because it transforms 1.7 fields to be represented as boolean (true/false/null) in graphql.

We could add a shortcut for 1.8, so we dont need to type out`HmisSchema::Enums::Hud::NoYesReasonsForMissingData
` each time. But now that we have the capability to select the enum based on a code, I'd rather do that if we are going to spend more time cleaning this up. In more detail:

Now that we `hud_lists.json`, we have the capability to map a code (`1.8`) to an enum `HmisSchema::Enums::Hud::NoYesReasonsForMissingData`. So we could add `code: "1.8"` into [here](https://github.com/greenriver/hmis-warehouse/blob/e003a33ae7ac309601185c6d0b1776a65784a944/app/models/concerns/hmis_structure/income_benefit.rb#L44-L46) and use that to set the enum automatically here using `hud_field`. I think that would preferable, to rely on the "source" representation as much as possible. Not sure if `hmis_configuration` is the right place for that, but I think its something we should consider